src/main/scala/amba/axi4/AsyncCrossing.scala

```diff
@@ -29,6 +34,11 @@ class AXI4AsyncCrossingSource(sync: Option[Int])(implicit p: Parameters) extends
   }
 }
 
+/**
+  * Sink(Manager) side for AXI4 crossing clock domain
```

Comment:
there is no Client-Manager concept in AXI. Just Master/Slave(according to spec of AXI) is enough.